templates/components/course-list/course-item-list.twig

```diff
@@ -48,11 +48,14 @@
         {%- endif -%}
   </div>
 
+  {%- if course_list_code|render_clean is not empty or course_list_academic_year|render_clean is not empty -%}
   <header class="su-course-list__header">
-    {%- if course_list_code is not empty -%}<strong>{{ course_list_code|render|striptags|replace({'\n': '', ' ': ''}) }}</strong>{%-  endif -%}&nbsp;|{%- if course_list_academic_year is not empty -%}{{ course_list_academic_year }}{%-  endif -%}
-
+    {%- if course_list_code is not empty -%}<strong>{{ course_list_code|render|striptags|replace({'\n': '', ' ': ''})}}</strong>{%-  endif -%}
```

Comment:
If the Academic Year field is filled in, the the `strong` tag is displaying as empty where the Course Code would be.

Reply:
Ah, the empty check needed the `render_clean` to really determine if it was empty.

Reply:
yeah, that did the trick.  
![image](https://user-images.githubusercontent.com/1263892/165633174-546385e7-40ce-419b-a9cf-bf9b6f0a7774.png)